internal/docs/release.md

```diff
@@ -16,13 +15,46 @@ To do a release:
         investigate; see the `prerelease.sh` script to see the exact command
         line to use.
     -   Run `./internal/testing/prerelease.sh cleanup` when you are done.
-    -   Commit the changes to the branch, and then delete the branch.
--   Identify the [commit](https://github.com/google/go-cloud/commits/master) you
-    want to label as the release; it should be a commit from around the end of
-    the last Sprint.
+    -   Delete the branch (`-D` to force deleting with uncommitted changes).
+
+-   Run the release helper tool to remove `replace` lines from the `go.mod`
+    files of submodules:
+
+    ```bash
+    $ go run internal/releasehelper/releasehelper.go dropreplace
+    ```
+
+    Check that everything looks in order (with `git diff`).
+
 -   Pick the new release name; it's probably `v0.x.0` where `x` is whatever the
     [last release](https://github.com/google/go-cloud/releases/latest) was plus
     one, but follow [semantic versioning](https://semver.org/).
+
+-   Run the release helper tool to set the version in `require` directives of
+    submodules to the new (yet unreleased) version:
+
+    ```bash
+    $ go run internal/releasehelper/releasehelper.go setversion v0.x.0
+    ```
+
+    Check that everything looks in order (with `git diff`).
+
+-   Commit and create a PR. Travis will fail for this PR because submodules
+    depend on a version of the main module that wasn't tagged yet. Enable force
+    merging in GitHub settings and force-merge the PR. Note that this does not
+    affect users, since a new version hasn't been tagged yet.
+
+-   Tag new versions for all released in Git (TODO(eliben): script this?):
```

Comment:
Yes please :-).

Reply:
Will send a followup change, likely to releasehelper